lib/include/hse_ikvdb/omf_version.h

```diff
@@ -70,7 +71,7 @@ enum {
 
 #define CNDB_VERSION           CNDB_VERSION13
 #define KBLOCK_HDR_VERSION     KBLOCK_HDR_VERSION5
-#define VBLOCK_HDR_VERSION     VBLOCK_HDR_VERSION2
+#define VBLOCK_FOOTER_VERSION  VBLOCK_FOOTER_VERSION3
```

Comment:
I would just use version 1. Maybe we can just reset all these versions and remove legacy stuff for v3.

Reply:
Yeah, we need to have that discussion. I'm inclined towards bumping the omf versions for HSE 3 rather than resetting it back to 1. Better debuggability that way.